### DISCUSSION TITLE
✅ DONE - RfC - replace mailchimp with a more cost-efficient tool

### DISCUSSION THREAD

**Discussion by macolo** [django-cms/django-cms#7980] [General]
- we currently use mailchimp with an estimated cost of 130 CHF per month (1500 CHF p.a.) to send out a quarterly newsletter to members, contributors and users who signed up for the django CMS demo
- we have around 8k contacts in mailchimp, growing at a rate of 1k per year.
- I see potential to reduce this cost to around 30 CHF per month and save the dCA significant money
- from a new service provider we need the following: unlimited contacts, multiple email lists and signup forms, a UI for simple email template buiding)
- alternatives I've checked so far are 
   - [Brevo (previously sendinblue)](https://www.brevo.com): unlimited contacts, 20k emails per month, lists, segments, signup forms and email template builder for 26 USD per month
   - [mailerlite](https://www.mailerlite.com/) (too expensive at ~70CHF per month)
   - [buttondown.com](https://buttondown.com/pricing): too expensive at 1400 CHF p.a.
- I would recommend  brevo - any thoughts?

**Comment by fsbraun** [comment]
I've used brevo. It seems fine unless you're using transactional emails (probably not a feature we need). When using transaction emails, occasionally I had issues with being declared a spammer despite only sending mails to addresses with double opt-in. This might be solved by now.

**Comment by marksweb** [comment]
I don't know too much about newsletter sending tools but I do know that Button Down is a popular one built on Django

https://buttondown.com/pricing

**Reply by jrief** [reply]
I wanted to propose the same. The latest Django Chat episode was about Buttondown: https://djangochat.com/episodes/buttondown-justin-duke

**Reply by jmduke** [reply]
:wave: I'm the founder of Buttondown (and also a very happy Django user)! 

Given that DSF is a non-profit, you all would qualify for our 50% non-profit discount — at 8K subscribers, you'd be looking at ~$395 USD/yr (or 33 CHF/mo). Happy to answer any questions you have either here or at justin@buttondown.email :)

**Comment by MacLake** [comment]
We use [Mailtrain](https://github.com/Mailtrain-org/mailtrain), JS based, open source (GPL 3.0), self-hosted: https://news.passiv.net/ A colleague of mine set it up, so I can’t tell you many details, but I can get in touch with him, if you have any questions.

**Comment by macolo** [comment]
Thank you everyone for contributing to this topic. After doing a thorough cost-benefit analysis we switched from MailChimp to Brevo.
